api/chat.go

```diff
@@ -53,7 +53,13 @@ func configGinChatRouter(router *gin.RouterGroup) {
 		if chat.Msg == "" || len(chat.Msg) > maxMessageLength {
 			return
 		}
-		if dao.IsUserCooledDown(fmt.Sprintf("%v", tumLiveContext.User.ID)) {
+		isCooledDown, err := dao.IsUserCooledDown(fmt.Sprintf("%v", tumLiveContext.User.ID))
+		if err != nil {
+			sentry.CaptureException(errors.New("could not determine whether user has cooled down"))
```

Comment:
Logrus can log the error and send an event to sentry as well (if configured).

```suggestion
			log.WithError(err).Error("could not determine whether user is cooled down")
```

Reply:
Ah, I always forget about that function... will change that!